Processing/BRCA_Deep_Dive.Rmd

```diff
@@ -2392,7 +2392,195 @@ kable(Well_Capacity)
 
 ```
 
-##### Fire sensitivity
+##### Fire 
+
+Contextual data (known fires, forested area):
+
+```{r}
+# HOW MANY KNOWN FIRES HAVE OCCURED?
+new_geomac <- sf::st_read("data/all/InterAgencyFirePerimeterHistory_All_Years_View/InterAgencyFirePerimeterHistory_All_Years_View.shp") 
+
+park_new_geomac <- new_geomac %>%
+  .[st_transform(park_boundary, crs = st_crs(new_geomac)),] 
```

Comment:
yep - this is the equivalent of st_filter() but faster. Since this is such a small filter anyways I went ahead and swapped them out to st_filter() like you suggested